### DISCUSSION TITLE
Help with Import Error

### DISCUSSION THREAD

**Discussion by divingmule** [python-pillow/Pillow#7161] [Q&A] [answered]
Hi, I'm getting an import error after installing Python 3.9 on Rpi OS Buster ->

`Traceback (most recent call last):
  File "/home/pi/TruckPi/muzic_player.py", line 12, in <module>
    from PIL import Image, ImageTk
  File "/usr/lib/python3.9/site-packages/PIL/Image.py", line 103, in <module>
    from . import _imaging as core
ImportError: /usr/lib/python3.9/site-packages/PIL/_imaging.cpython-39-arm-linux-gnueabihf.so: undefined symbol: opj_encoder_set_extra_options`

Can anyone help?
Thanks in advance.

**Comment by radarhere** [comment, marked as answer]
Hi. Is anything in https://github.com/python-pillow/Pillow/issues/7067 helpful?

**Reply by divingmule** [reply]
Yes, very much so.

Thank you!
